src/NoCodes.ts

```diff
@@ -0,0 +1,37 @@
+import NoCodesApi from "./NoCodesApi";
+import NoCodesConfig from "./NoCodesConfig";
+import NoCodesInternal from "./internal/NoCodesInternal";
+
+export default class NoCodes {
+  private constructor() {}
+
+  private static backingInstance: NoCodesApi | undefined;
+
+  /**
+   * Use this variable to get a current initialized instance of the NoCodes SDK.
```

Comment:
Let's everywhere in comments use our public naming - No-Codes
```suggestion
   * Use this variable to get a current initialized instance of the No-Codes SDK.
```